frontend/src/tests/pages/RiderDashboard/RideDashboardIndexPage.test.js

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from "@testing-library/react";
+import RideDashboardIndexPage from "main/pages/RiderDashboard/RideDashboardIndexPage";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+
+import { apiCurrentUserFixtures } from "fixtures/currentUserFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+
+
+describe("RideDashboardIndexPage tests", () => {
+
+    const axiosMock = new AxiosMockAdapter(axios);
+
+    const setupUserOnly = () => {
+        axiosMock.reset();
+        axiosMock.resetHistory();
+        axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+        axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingNeither);
+    };
+
+    const queryClient = new QueryClient();
+    test("Renders expected content", () => {
+        // arrange
+
+        setupUserOnly();
```

Comment:
You will need to add an apiCurrentUserFixture that has the role `DRIVER` and a test to make sure that the user with the role DRIVER can get the expected content on the page, and that the user that does NOT have the `DRIVER` role can NOT get that page.